tango_with_django_project/urls.py

```diff
@@ -21,6 +23,7 @@
 urlpatterns = [
     path('', views.index, name='index'),
     path('rango/', include('rango.urls')),
+    path('rango2/', views.training, name='training'),
```

Comment:
Please change the naming of 'rango2/' to more verbose. It should understandable from the name, what it is for.

You can change it to 'training' in this case.